daemon/config/README.md

```diff
@@ -46,7 +46,7 @@ Please note that your data history will be lost if you have modified `history` p
 | glibc malloc arena max for plugins|`1`|See [Virtual memory](../#virtual-memory).|||
 | glibc malloc arena max for Netdata|`1`|See [Virtual memory](../#virtual-memory).|||
 | hostname|auto-detected|The hostname of the computer running Netdata.|||
-| history|`3996`|The number of entries the `netdata` daemon will by default keep in memory for each chart dimension. This setting can also be configured per chart. Check [Memory Requirements](../../database/#database) for more information.|||
+| history|`3996`| Deprecated in favor of `memory mode = dbengine` and `page cache size`/`dbengine disk space` for long-term metrics storage. Only works with `memory mode = save/map`, and changes the number of metrics entries Netdata will store for each chart.|||
```

Comment:
There are some issues with this statement. First of all, it doesn't only work with save/map. It also works with alloc/ram. Secondly, it's not deprecated. Maybe we'll deprecate save/map, but we don't have any plan to deprecate ram which is also controlled by history.

Reply:
Good points. I changed this to say that it only works with those 4 modes, not dbengine.